go.mod

```diff
@@ -260,15 +258,12 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	lukechampine.com/blake3 v1.2.1 // indirect
-	modernc.org/libc v1.22.3 // indirect
-	modernc.org/mathutil v1.5.0 // indirect
-	modernc.org/memory v1.5.0 // indirect
-	modernc.org/sqlite v1.21.1 // indirect
+	modernc.org/libc v1.37.6 // indirect
+	modernc.org/mathutil v1.6.0 // indirect
+	modernc.org/memory v1.7.2 // indirect
+	modernc.org/sqlite v1.28.0 // indirect
 	nhooyr.io/websocket v1.8.7 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
 replace github.com/onflow/flow-cli/flowkit => ./flowkit
-
-// this is a woraround for the flow-go/crypto cross compliation issue
-replace github.com/onflow/flow-go/crypto => github.com/onflow/flow-go/crypto v0.24.9
```

Comment:
A similar `replace` is still needed if you are planning to release a version from this branch.  The reason is that the CLI build process didn't get updated to manage cross compilation yet (like what I did for the [emulator](https://github.com/onflow/flow-emulator/pull/548)).

In this PR, The statement can become:
``` replace github.com/onflow/crypto => github.com/onflow/crypto v0.24.9```

If no release is needed then it's fine to remove this statement.

Reply:
Thanks for having a look! I've added the replace statement back in ab46a892db1de6b8e74bc29b8eb738e3e710b21c